packages/taquito/src/wallet/wallet.ts

```diff
@@ -257,6 +258,37 @@ export class Wallet {
     });
   }
 
+  /**
+   *
+   * @description failing_noop operation that is guaranteed to fail.
+   *
+   * @returns Signature for a failing_noop
+   *
+   * @param params operation parameter
+   */
+  async signFailingNoop(params: WalletFailingNoopParams) {
+    const failingOperation = await this.context.prepare.failingNoop(params);
+    const forgeable = this.context.prepare.toForge(failingOperation);
```

Comment:
like above when prepare already returning an object ready to forge that there is no need of .toForge(). Can be discussed